src/uwtools/tests/config/test_jinja2.py

```diff
@@ -186,7 +186,7 @@ def test_register_filters_path_join(key):
 
 
 def test_render(values_file, template_file, tmp_path):
-    outfile = str(tmp_path / "out.txt")
+    outfile = tmp_path / "out.txt"
```

Comment:
Unnecessary, and downstream code expects a `Path` anyway. Ditto for the diff just below.